packages/oruga-next/src/components/tooltip/Tooltip.vue

```diff
@@ -302,6 +306,12 @@ const rootClasses = defineClasses(
         null,
         computed(() => !!props.teleport),
     ],
+    [
+        "rootPositionClass",
```

Comment:
I don't understand the need for this class? By setting `teleport` to true, the actual tooltip HTMLElement is no longer below the root element. As a result, the root element doesn't need to have any visual styling.

Reply:
I added this class to replicate how the classes worked in the last version. The [current styles](https://github.com/whiplashwebb/theme-bulma/blob/181835d3d6d55ec7fc95fbaabfa8a6125090796a/src/assets/scss/components/_tooltip.scss#L108) assume the position class will be on the root. It looks like I could update these styles pretty easily to change that assumption, but that seems more risky than replicating the old behaviour. Other devs may have built on top of this assumption and I don't want to disrupt that unless there's a solid reason. 

I didn't consider the impact of the teleport prop. That one is new to me. Wouldn't teleporting the tooltip contents break tooltip positioning? It needs the position:relative on the root to absolutely position the contents. Is there an example of this usage somewhere I can test against?

Reply:
I understand what the teleport prop does here, I'm trying to understand the real-world usage of this prop. Is there an example I can look at in the other themes? 

Curiosity about teleport aside, I don't see the harm in adding this class. Your point about having styles on the tooltip content is valid, but we still have the output of `positionClass` in `contentClasses`, so this shouldn't break anything and the class can always be disabled in other themes if it's not desired.

Reply:
But does it work for people using the Bulma theme and the teleport prop?

For example, if you use teleport, the tooltip will not refer to a container with overflow hidden or something like that, where it would be cut off.

Reply:
that's a totally valid point, i really don't know the answer. that's why i'm asking for an example of usage, i don't understand what "working" would even look like with teleport. Bulma does have this feature, neither does Buefy, it's new to Oruga. Is there an example of this in action from the world of bootstrap? i used to do a ton of bootstrap, but it's been years at this point.

Reply:
Within vanilla bulma this just wouldn't work. There's a parent/child relationship intentionally written into the code. We could certainly add something to Oruga/Bulma to support this, but it would be custom code on top of Bulma, and I would argue it should be tackled as a separate feature.

Reply:
nvm. the root container classes get used to the teleport container too. So no worry. However, i think you can adjust the theme css to use the aktuall normal positionClass prop. I don't see the need for the root one looking at the theme code.